lib/core/network/repository/auth_repository.dart

```diff
@@ -85,6 +92,16 @@ class AuthRepository {
     _appSharedPrefs.setUserProfileData(userProfileData);
     _secureStorageService.setUserAuthData(userAuthData);
 
+    /// Get firebase device token and save to firebase
+    _firebasePushNotificationsService.getDeviceToken().then(
```

Comment:
when you log in or create account, the token is saved into firebase with your account